23_remove_last_occurrence_of_character.js

```diff
@@ -0,0 +1,32 @@
+// Write a javaScript program to remove last occurrence of a character in a given string.
+
+
+let text = {
+    string: "My Name is Vasu",
+    search: "a",
+    count: 0
+}
+
+for (let i = 0; i < text.string.length; i++) {
+
+    if (text.string[i] == text.search) {
```

Comment:
Told you already, dont drink and code

Reply:
I have understood your suggestion and updated this code.
Thank you